exercicioLogica16/src/main/java/org/example/Emprestimo.java

```diff
@@ -4,33 +4,31 @@
 import java.util.ArrayList;
 
 public class Emprestimo {
-    private Livro livro;
 
     private Pessoa pessoa;
 
     private ArrayList<Livro> listaLivros;
 
     public Emprestimo(Livro livro, Pessoa pessoa) {
```

Comment:
Livro pedido no construtor não é utilizado abaixo.
Dependendo da implementação tanto um livro (como no construtor atual) quanto uma lista de livros poderia ser informado.

No caso de uma lista já ser informado ao criar o emprestimo o metodo poderia ser:

  public Emprestimo(ArrayList<Livro> listaLivros, Pessoa pessoa) {
        this.pessoa = pessoa;
        this.listaLivros = listaLivros;
    }